DesignRules.md

```diff
@@ -632,30 +648,233 @@ Changes in APIs are inevitable. APIs should therefore always be versioned, facil
 
 ## Transport Security
 
-Transport security is essential to safeguard the confidentiality, integrity, and authenticity of data during its transmission.
+This section describes security principles, concepts and technologies to apply when working with APIs.
+Controls need to be applied for the security objectives of integrity, confidentiality and availability of the API and services and data provided thereby.
+The [architecture section of the API strategy](https://docs.geostandaarden.nl/api/API-Strategie-architectuur/) contains architecture patterns for implementing Transport security.
 
-<div class="rule" id="/core/transport-security" data-type="technical">
-  <p class="rulelab">Apply the transport security module</p>
-  <dl>
-    <dt>Statement</dt>
-    <dd>
-      The [[[ADR-TS]]] MUST be applied.
-    </dd>
-    <dt>Rationale</dt>
-    <dd>
-      The [[[ADR-TS]]] formalizes three rules to apply to APIs:
-      <ol>
-        <li>Secure connections using TLS</li>
-        <li>No sensitive information in URIs</li>
-        <li>Use CORS to control access</li>
-      </ol>
-      Furthermore, the module describes best practices for security headers, browser-based applications, and other HTTP configurations. These best practices MUST be considered and the considerations SHOULD be published in the API documentation. Transport security is the baseline for REST API resources and the data concerned is a vital asset of the government. The rules and best practices are considered the minimal security principles, concepts and technologies to apply.
-    </dd>
-    <dt>Implications</dt>
-    <dd id="implications"></dd>
-  </dl>
+The scope of this section is limited to generic security controls that directly influence the visible parts of an API.
+Effectively, only security standards directly applicable to interactions are discussed here.
+
+In order to meet the complete security objectives, every implementer MUST also apply a range of controls not mentioned in this section.
+
+Note: security controls for signing and encrypting of application level messages will be part of a separate extension, [Signing and Encryption](https://geonovum.github.io/KP-APIs/API-strategie-modules/encryption/).
+
+<span id="api-11"></span>
+<div class="rule" id="/core/transport/tls" data-type="technical">
+  <p class="rulelab">Secure connections using TLS</p>
+   <dl>
+      <dt>Statement</dt>
+      <dd>
+         <p>One should secure all APIs assuming they can be accessed from any location on the internet. Information MUST be exchanged over TLS-based secured connections. No exceptions, so everywhere and always. This is <a href="https://wetten.overheid.nl/BWBR0048156/2023-07-01">required by law</a>.
+         <p>One SHOULD follow the latest NCSC guidelines [[NCSC.TLS]]
```

Comment:
Andere documenten hebben al een ref naar dit document, waardoor het reeds in de algemene lijst staat onder [[NCSC 2021]] (vergeef de spatie):
https://github.com/Logius-standaarden/publicatie/blob/fd2b345c75b537af812de5ffa419edeedfa578a3/respec/organisation-config.js#L388-L392